README.adoc

```diff
@@ -136,6 +136,7 @@ Translations of the guide are available in the following languages:
 * https://github.com/Nondv/clojure-style-guide/blob/master/ru/README.md[Russian]
 * https://github.com/jeko2000/clojure-style-guide/blob/master/README.md[Spanish]
 * https://github.com/LeaveNhA/clojure-style-guide/blob/master/README.adoc[Turkish]
+* https://github.com/ildivan/clojure-style-guide/blob/ita/README.adoc[Italian] (Under progress)
```

Comment:
This list is sorted alphabetically, so please move it up to the second spot. (should be between Chinese and Japanese)